F.R.O.S.T/app/src/main/java/t/s/o/r/f/frost/MainActivity.java

```diff
@@ -65,6 +66,8 @@ protected void onCreate(Bundle savedInstanceState) {
         tv = (TextView) findViewById(R.id.collision_text); //TextView for collision text.
         v = findViewById(R.id.cc_view4);
         tvbg = findViewById(R.id.text_background);
+        fireImage = (ImageView)findViewById(R.id.imageView1);
```

Comment:
Maybe have a space here? If they are not related to the stuff above. Maybe a comment/description